apisix/utils/log-util.lua

```diff
@@ -64,7 +64,11 @@ local function get_full_log(ngx, conf)
         else
             local body_file = ngx.req.get_body_file()
             if body_file then
-                log.request.body_file = body_file
+                local lines = {}
```

Comment:
https://stackoverflow.com/questions/11201262/how-to-read-data-from-a-file-in-lua

Read the file content in this way, is the right way:

```lua
local open = io.open

local function read_file(path)
    local file = open(path, "rb") -- r read mode and b binary mode
    if not file then return nil end
    local content = file:read "*a" -- *a or *all reads the whole file
    file:close()
    return content
end

local fileContent = read_file("foo.html");
print (fileContent);
```

Reply:
`if not file then return nil end`
do we need this part of the code? if the body is not present then can we not assume the data will be present in the file?